.circleci/config.yml

```diff
@@ -154,14 +164,14 @@ workflows:
 #              only: /.*/
       # END - Test legacy migrations
 
-      - golangci/lint:
+      - validate:
           filters:
             tags:
               only: /.*/
       - nancy/test:
-        filters:
-          tags:
-            only: /.*/
+          filters:
+            tags:
+              only: /.*/
```

Comment:
Noticed that the indentation was wrong here because of validation errors reported by goland